src/scripts/scripts.js

```diff
@@ -0,0 +1,825 @@
+/* global event, localStorage, document, window */
+/* eslint max-len: ["error", { "code": 190 }] */
+
+const Keyboard = {
```

Comment:
It's better to move this to separate config file

Reply:
Comment with global in it, I can't fix, eslint throw error. Next comment move to config file